Microsoft.Toolkit.Uwp.SampleApp/Microsoft.Toolkit.Uwp.SampleApp.csproj

```diff
@@ -176,6 +176,7 @@
     <Content Include="Assets\People\vlad.png" />
     <Content Include="Assets\Photos\BigFourSummerHeat.jpg" />
     <Content Include="Assets\Photos\BisonBadlandsChillin.jpg" />
+    <Content Include="Assets\Photos\CameraSource.png" />
```

Comment:
Use a font icon instead and remove image

Reply:
Done.